### PR TITLE
Added RemoveUSB

### PR DESCRIPTION
**Pull request by jonasw234**
Small plugin to detect removable USB devices and provide a query list to eject them safely.

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -1426,5 +1426,17 @@
...
+        "ID":"5DBA462B9AB77AC5DC158EB5047367F0",
+        "Name":"RemoveUSB",
+        "Description":"Removes USB drives",
+        "Author":"Jonas A. Wendorf",
+        "Version":"1.0.0",
+        "Language":"python",
+        "Website": "https://github.com/jonasw234/FlowLauncher.Plugin.RemoveUSB",
+        "IcoPath": "https://cdn.statically.io/gh/jonasw234/FlowLauncher.Plugin.RemoveUSB/main/icon/usb_stick.png",
+        "UrlSourceCode": "https://github.com/jonasw234/FlowLauncher.Plugin.RemoveUSB",
+        "UrlDownload": "https://github.com/jonasw234/FlowLauncher.Plugin.RemoveUSB/releases/download/1.0.0/FlowLauncher.Plugin.RemoveUSB.zip",
```
Comment (Garulf) [inline plugins.json:1440]:
```suggestion
        "UrlDownload": "https://github.com/jonasw234/FlowLauncher.Plugin.RemoveUSB/releases/download/1.0.0/FlowLauncher.Plugin.RemoveUSB.zip"
```

**Review by Garulf — CHANGES_REQUESTED** [review]

**Comment by jonasw234** [comment]
I removed the superfluous comma from the `plugin.json` in the repo and the release as well (same as your edit). Is there anything else I need to do to re-run the tests?

**Comment by jonasw234** [comment]
Ah, my bad, I had the comma in the wrong line. Now it will hopefully work. Sorry for the mess.

**Comment by Garulf** [comment]
Looks like the run test is failing because of a missing library. Please include all external libraries in your zip file. 

In this case it looks like the FlowLauncher library is missing.

**Review by Garulf — CHANGES_REQUESTED** [review]
Looks like the run test is failing because of a missing library. Please include all external libraries in your zip file. 

In this case it looks like the FlowLauncher library is missing.

**Comment by jonasw234** [comment]
Ah, I thought that FlowLauncher would be included automatically by your tests. Added it. Thanks for your patience :)

**Comment by Garulf** [comment]
Please update the PR with the new version.

**Comment by jonasw234** [comment]
I added the Flow Launcher package to the `lib` folder like I in the plugin [isPrime](https://github.com/lvonkacsoh/Flow.Launcher.Plugin.IsPrime). Is anything else needed?

**Comment by Garulf** [comment]
The test tries to emulate Flow Launcher using its built-in embedded Python which most users will probably be using. It will not see package directories like normal Python.

Please see here for more information: https://www.flowlauncher.com/docs/#/py-develop-plugins

**Comment by jonasw234** [comment]
I think I got it now. At least the workflow action worked and I added the additional lines in the documentation.

File: plugins.json [hunk 2]
```diff
@@ -1426,5 +1426,17 @@
...
+        "ID":"5DBA462B9AB77AC5DC158EB5047367F0",
+        "Name":"RemoveUSB",
+        "Description":"Removes USB drives",
+        "Author":"Jonas A. Wendorf",
+        "Version":"1.0.0",
+        "Language":"python",
+        "Website": "https://github.com/jonasw234/FlowLauncher.Plugin.RemoveUSB",
+        "IcoPath": "https://cdn.statically.io/gh/jonasw234/FlowLauncher.Plugin.RemoveUSB/main/icon/usb_stick.png",
+        "UrlSourceCode": "https://github.com/jonasw234/FlowLauncher.Plugin.RemoveUSB",
+        "UrlDownload": "https://github.com/jonasw234/FlowLauncher.Plugin.RemoveUSB/releases/download/1.0.0/FlowLauncher.Plugin.RemoveUSB.zip"
```
Comment (Garulf) [inline plugins.json:1440]:
```suggestion
        "UrlDownload": "https://github.com/jonasw234/FlowLauncher.Plugin.RemoveUSB/releases/download/v1.0.0/FlowLauncher.Plugin.RemoveUSB.zip"
```
Test is unable to find release download.
New versioning has "v" prepended

**Comment by Garulf** [comment]
Looks like it passed the test! 🥳 

Looks like the test itself failed somehow though.

**Review by Garulf — APPROVED** [review]

**Review by Garulf — CHANGES_REQUESTED** [review]
Python 3.8 doesn't support `dict` usage as a type hint. In order to support the minimal version FLow Launcher supports please use `Dict` from the `typing` module on this line:

https://github.com/jonasw234/FlowLauncher.Plugin.RemoveUSB/blob/e3dca7a8e3410b054d5f74bf6ea8a8e19200bdaf/removeusb.py#L50

**Comment by jonasw234** [comment]
Ah, changed it, thanks!

**Comment by Garulf** [comment]
I should've checked the rest of the code. Looks like you’re using `list` as well. Please use `List` the same as `Dict` earlier.

**Comment by jonasw234** [comment]
Okay, I just changed it, tested again with Python 3.8.10 and it seems to have worked. Fingers crossed.

**Comment by Garulf** [comment]
It passed! (For real this time) 🥳

**Review by Garulf — APPROVED** [review]

**Comment by jonasw234** [comment]
Awesome! Thank you so much for your support, you’re amazing!

**Pull request merged**